src/picknik_dual_ur_arm_config/config/moveit/left_ur_servo.yaml

```diff
@@ -0,0 +1,60 @@
+###############################################
+# Modify all parameters related to servoing here
+###############################################
+
+# Enable dynamic parameter updates
+enable_parameter_update: true
+
+## Properties of incoming commands
+command_in_type: "unitless" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
+scale:
+  # Scale parameters are only used if command_in_type=="unitless"
+  linear: 2.0  # Max linear velocity. Meters per publish_period. Unit is [m/s]. Only used for Cartesian commands.
+  rotational: 3.0 # Max angular velocity. Rads per publish_period. Unit is [rad/s]. Only used for Cartesian commands.
+  # Max joint angular/linear velocity. Rads or Meters per publish period. Only used for joint commands on joint_command_in_topic.
+  joint: 2.0
+
+# Default to driving the arm at 50% maximum speed.
+override_velocity_scaling_factor: 0.5
+
+## Properties of outgoing commands
+publish_period: 0.002  # 1/Nominal publish rate [seconds]
+
+# What type of topic does your robot driver expect?
+# Currently supported are std_msgs/Float64MultiArray or trajectory_msgs/JointTrajectory
+command_out_type: std_msgs/Float64MultiArray
```

Comment:
Would probably be good to use the JointTrajectory output type here + a JTC instead of a streaming controller, like in the other UR configs since @pac48 's changes.

Using a JTC also makes it easier to transition to doing servo with the mega-JTC for both arms in the near future.